src/modules/components/asset/assetDataListItem/assetDataListItemStructure/assetDataListItemStructure.tsx

```diff
@@ -0,0 +1,100 @@
+import classNames from 'classnames';
+import type React from 'react';
+import { useMemo } from 'react';
+import { Avatar, DataList, NumberFormat, Tag, formatterUtils, type IDataListItemProps } from '../../../../../core';
+
+export interface IAssetDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The logo source of the asset
+     */
+    logoSrc?: string;
+    /**
+     * The name of the asset.
+     */
+    name?: string;
+    /**
+     * The symbol of the asset.
+     */
+    symbol?: string;
+    /**
+     * The amount of the asset.
+     */
+    amount?: number | string;
+    /**
+     * The fiat price of the asset.
+     */
+    fiatPrice?: number | string;
+    /**
+     * the price change in percentage of the asset (E.g. in last 24h).
+     */
+    priceChange?: number;
+}
+
+export const AssetDataListItemStructure: React.FC<IAssetDataListItemStructureProps> = (props) => {
+    const { logoSrc, name = '-', amount = 0, symbol, fiatPrice = 0, priceChange = 0, ...otherProps } = props;
+
+    const usdAmountChanged = useMemo(() => {
+        const usdAmount = (amount ? Number(amount) : 0) * (fiatPrice ? Number(fiatPrice) : 0);
```

Comment:
As we're setting the default values, we can simplify these calculations. I would also use the JS native BigInt to make these calculations, like:
```suggestion
        const usdAmount = BigInt(amount) * BigInt(fiatPrice)
```